Formula/tape-cli.rb

```diff
@@ -0,0 +1,21 @@
+require "language/node"
+
+class TapeCli < Formula
+  desc "CLI for mobile devs - share screenshots, recordings as video or GIFs"
+  homepage "https://tape.sh/"
+  url "https://registry.npmjs.org/tape-cli/-/tape-cli-0.10.2.tgz"
+  sha256 "050a659b94d23ba50de17fd335c6155dd41422cc99a443c5c93a027f5dc70c0b"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+    ohai "🎉 Please run `tape config --setup` to get started \n"
+  end
+
+  test do
+    system "#{bin}/tape", "help"
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Hi @SMillerDev, updated the PR as per your comment.